staging/copy.sh

```diff
@@ -60,6 +60,12 @@ function mkcp() {
     mkdir -p "${CLIENT_REPO_TEMP}/$2" && cp -r "${MAIN_REPO}/$1" "${CLIENT_REPO_TEMP}/$2"
 }
 
+# save copies code from client-go into the temp folder to make sure we don't lose it by accident
+# TODO this is temporary until everything in certain directories is authoritative
+function save() {
```

Comment:
> I see. So you want to track some files in /staging as canonical, while some are still programmatically generated from elsewhere? This is going to be super confusing. Can you add an explanatory comment near the top of this file, that there are two classes of things in the staging directory?

Ok.  I do see this as mostly intermediate. Eventually the staging copy is just API types and we simply regenerate the clients.